src/signalrclient/stdafx.cpp

```diff
@@ -2,4 +2,8 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-#include "stdafx.h"
\ No newline at end of file
+#include "stdafx.h"
+
+#ifdef OVERRIDE_NEW_DELETE_HEADER
+#include STRINGIFY(OVERRIDE_NEW_DELETE_HEADER)
+#endif
```

Comment:
I could add any header using this, right? What do you think about calling this `INJECT_HEADER` instead of `OVERRIDE_NEW_DELETE_HEADER`?

Reply:
You could, but we'll be adding another one of these that needs to go in stdafx.h in order to add some `#define`s for WPP logging.

Reply:
How about `INJECT_HEADER_AFTER_STDAFX`?

Reply:
I could live with that 👍